.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: python
+python:
+    - "2.7"
+before_install:
+    - "export DISPLAY=:99"
+    - "sh -e /etc/init.d/xvfb start"
+install:
+    - "pip install -e git://github.com/edx/xblock-sdk.git#egg=xblock-sdk"
+    - "pip install -r requirements.txt"
+    - "pip install -r test-requirements.txt"
+    - "pip install -r $VIRTUAL_ENV/src/xblock-sdk/requirements.txt"
+    - "pip install -r $VIRTUAL_ENV/src/xblock-sdk/test-requirements.txt"
```

Comment:
@e-kolpakov You might want to add a line similar to https://github.com/open-craft/problem-builder/blob/48fd595279493988393ef7dbbad4c228634dc304/.travis.yml#L12 to ensure this is installed the same was as in production -- currently it is installed with the `pip -e` option, which uses the development mode, and doesn't catch missing assets.

When installed with `python setup.py sdist && pip install dist/xblock-group-project-v2-0.2.tar.gz` it currently gives https://gist.github.com/antoviaque/1f7b051da6986ed9a256